src/services/index.ts

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
+import axios from "axios";
+
+const defaultConfig = {
+  "Content-Type": "application/json",
+  "Access-Control-Allow-Origin": "*",
+};
+
+const api = axios.create({
+  ...defaultConfig,
+  baseURL: "http://localhost:8000",
```

Comment:
get this url from env variable, so, it will be possible to use the correct url for staging and production environments